binaries/scheduler/main.go

```diff
@@ -7,35 +7,30 @@ import (
 	"github.com/apache/thrift/lib/go/thrift"
 	"github.com/scootdev/scoot/sched/config"
 	"github.com/scootdev/scoot/scootapi/server"
-	"golang.org/x/net/context"
 )
 
 var addr = flag.String("addr", "localhost:9090", "Bind address for api server.")
-var cfgFile = flag.String("cfg", "", "Path to sched overlay config file.")
+var cfgText = flag.String("sched_config", "", "Scheduler Configuration.")
```

Comment:
Yeah, good question.

1) It's not super hard in a shell script to pass an arbitrarily-large flag.
2) It is harder to have to create a file just to hold something
3) I agree. If you pass "file:foo.txt" it should read from foo.txt. I put a TODO about this in config.go (at the top of the function Parse)